warehouse/query-core/src/main/java/datawave/query/jexl/lookups/BoundedRangeIndexLookup.java

```diff
@@ -50,7 +51,7 @@ public class BoundedRangeIndexLookup extends AsyncIndexLookup {
     private final LiteralRange<?> literalRange;
 
     protected Future<Boolean> timedScanFuture;
-    protected long lookupStartTimeMillis = Long.MAX_VALUE;
+    protected AtomicLong lookupStartTimeMillis = new AtomicLong(Long.MAX_VALUE);
```

Comment:
I am guessing you did this because of thead access?  Could have used the "volatile" keyword as well.